akka-distributed-data/src/main/scala/akka/cluster/ddata/ORMultiMap.scala

```diff
@@ -3,22 +3,26 @@
  */
 package akka.cluster.ddata
 
-import akka.cluster.{ UniqueAddress, Cluster }
+import akka.cluster.{ Cluster, UniqueAddress }
 import akka.annotation.InternalApi
+import akka.cluster.ddata.ORMap._
 
 object ORMultiMap {
 
-  val _empty: ORMultiMap[Any, Any] = new ORMultiMap(ORMap.empty)
+  val _empty: ORMultiMap[Any, Any] = new ORMultiMap(ORMap.empty, VanillaORMultiMapTag)
+  val _emptyWithDeltaDelta: ORMultiMap[Any, Any] = new ORMultiMap(ORMap.empty, DeltaDeltaORMultiMapTag)
   /**
    * Provides an empty multimap.
    */
   def empty[A, B]: ORMultiMap[A, B] = _empty.asInstanceOf[ORMultiMap[A, B]]
+  def emptyWithDeltaDelta[A, B]: ORMultiMap[A, B] = _emptyWithDeltaDelta.asInstanceOf[ORMultiMap[A, B]]
```

Comment:
document the difference compared to `empty` including the tradefoffs (growing state etc)